README.md

```diff
@@ -10,6 +9,27 @@
 - JS
 - CSS
 - VScode
+- Webpack
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.For SSH:`git@github.com:NatiG25/ToDoList.git`For https:`https://github.com/NatiG25/ToDoList.git`
+
+### Install
+Run `npm install` after cloning the repo.
+
+### Usage
+Run `npm start` for development. Run `npm run build` for production. 
+
+### Run tests
+- Run `npx eslint .` for JS check
+- Run `npx eslint . --fix` to fix JS errors if found
+- Run `npx stylelint "**/*.{css,scss}"` for CSS check
+- Run `npx stylelint "**/*.{css,scss}" --fix` to fix CSS errors if found.
+
+## Live Demo
+
+[Live Demo Link](http://localhost:8080/)
```

Comment:
- [ ] The live demo link is broken. I would recommend deploying your project and adding the online link to your README.
- [ ]  The `MIT link` is also broken in the README file. I would highly recommend fixing those links to make README professional.